backend/api/models.py

```diff
@@ -39,7 +39,7 @@ class BaseCourseBuildSettings(BaseModel):
     instructor_firstname: str
     instructor_lastname: str
     instructor_email: str
-    school_district: str
+    school_district_name: str
```

Comment:
I'm assuming we're doing this name change for clarity, correct? (e.g. in case we ever want to also expose `school_district_id`)

Reply:
That's correct.